libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -39,6 +42,7 @@ data class NavigationOptions(
     val isFromNavigationUi: Boolean,
     val isDebugLoggingEnabled: Boolean,
     val deviceProfile: DeviceProfile,
+    val rerouteAdapter: RouteRefreshAdapter,
```

Comment:
Route refresh isn't a required feature for navigation. Could we remove it from `NavigationOptions`? That way we can reserve it for required configuration